perf/CMakeLists.txt

```diff
@@ -51,7 +54,6 @@ function(create_perf_test_target)
                         "${singleValues}"
                         "${multiValues}"
                         ${ARGN})
-  message(STATUS "Creating C performance test ${PERF_TARGET}_perftest")
```

Comment:
Why do you drop this message?

Reply:
The same as above:

Now CMake prints a message for every generated unit/perf test. I like how we did in LuaJIT - print one message for each testsuite. So I did the same in this patchset. This line is moved to another place.